diff --git a/drivers/misc/Kconfig b/drivers/misc/Kconfig
index 4a01031..5e765d9 100644
--- a/drivers/misc/Kconfig
+++ b/drivers/misc/Kconfig
@@ -515,6 +515,14 @@ config SRAM
 	  the genalloc API. It is supposed to be used for small on-chip SRAM
 	  areas found on many SoCs.
 
+config HD44780
+        tristate "HD44780 LCD controller bit-banged driver"
+        depends on ARM
+        default m if ARM
+        help
+          Driver for HD44780 compatible LCD controllers. All pin and screen
+          information is loaded from the device tree.
+
 source "drivers/misc/c2port/Kconfig"
 source "drivers/misc/eeprom/Kconfig"
 source "drivers/misc/cb710/Kconfig"
